src/components/button.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+
+class Button extends Component{
+//set up constructor and super, with state object with two keys
+//one key offOn and one key bulb, to show the color of the bulb at the off or on state
+  constructor(props) {
+    super(props);
+    this.state = {
+      offOn: 'off',
+      bulb: 'white'
+    }
+  }
+//code a function that toggles offOn and bulb using conditionals
+
```

Comment:
You could use a ternary here, but I also like to type out my if(else) statements. Makes it easier to read imo